python/.flake8

```diff
@@ -11,8 +11,18 @@ extend-ignore =
     # 'from module import *' used; unable to detect undefined names (F403)
     F403
```

Comment:
Is this really required? This type of import is bad form, I'd like to refactor it if we can

Reply:
Do we know which file fails this check?

Reply:
Removed the config and violations